extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/ContractAgreementResult.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Schema(description = "EDC-status-defining Contract-Agreementss")
```

Comment:
Beim Self-Review extra auf die Dto-Klassen und Endpunkte achten.

Ein mal jede Klasse / Endpunkt / Feld auf Doku untersuchen. 

Hier wäre die Richtige Dokumentation "All data required for our UI's Contract Agreement Page." Keine unnötigen Wiederholungen von EDC